components/automate-ui/src/app/pages/team-management/team-management.component.ts

```diff
@@ -83,10 +91,27 @@ export class TeamManagementComponent implements OnInit, OnDestroy {
     this.store.dispatch(new GetTeams());
 
     this.iamMajorVersion$
-    .pipe(takeUntil(this.isDestroyed))
-    .subscribe((version) => {
-      if (version === null) { return; }
-        this.isV1 = version === 'v1';
+      .pipe(takeUntil(this.isDestroyed))
+      .subscribe((majorVersion) => {
+        if (majorVersion === null) { return; }
+          this.isMajorV1 = majorVersion === 'v1';
+        });
+
+    this.iamMinorVersion$
+      .pipe(takeUntil(this.isDestroyed))
+      .subscribe((minorVersion) => {
+        if (minorVersion === null) { return; }
+          this.isMinorV1 = minorVersion === 'v1';
+        });
+
+    this.store.select(projectsAssignable)
+      .subscribe((assignable: ProjectsFilterOption[]) => {
+        this.dropdownProjects = assignable.map(p => {
+          return <Project>{
+            id: p.value,
+            name: p.label
+          };
```

Comment:
Essentially same point I mentioned earlier. To be a `Project` object, it must also have a type field. But, of course, the UI does not need to care about that field. So is it worth having a `RenderedProject` object that is a subset without the type? 🤷‍♂ I am on the fence.

Reply:
Seems more straightforward to treat it as a full project, if we just have to pass around one extra field. Though if we're not getting `type` back from the API that would change my answer 🤔